proc/router.go

```diff
@@ -0,0 +1,203 @@
+package proc
+
+import (
+	"sync"
+
+	"github.com/brimdata/zed/zbuf"
+)
+
+type Selector interface {
+	Forward(*Router, zbuf.Batch) error
```

Comment:
```suggestion
type Forwarder interface {
	Forward(*Router, zbuf.Batch) error
```

Reply:
Or, if you prefer Selector.
```suggestion
type Selector interface {
	Select(*Router, zbuf.Batch) error
```